src/ZoneServer/Network/Send.cs

```diff
@@ -4384,6 +4431,39 @@ public static void ZC_KNOCKDOWN_INFO(ICombatEntity entity, ICombatEntity target,
 			entity.Map.Broadcast(packet, entity);
 		}
 
+		/// <summary>
+		/// Display an effect on the floor for nearby players
+		/// </summary>
+		/// <param name="entity"></param>
+		/// <param name="packetString"></param>
+		/// <param name="position"></param>
+		/// <param name="f1"></param>
+		/// <param name="f2"></param>
+		/// <param name="f3"></param>
+		/// <param name="f4"></param>
+		/// <param name="f5"></param>
+		public static void ZC_GROUND_EFFECT(ICombatEntity entity, string packetString, Position position, float f1, float f2, float f3, float f4, float f5)
+		{
+			if (!ZoneServer.Instance.Data.PacketStringDb.TryFind(packetString, out var packetStringData))
```

Comment:
Reminder that we have a helper extension for packet strings now, `Packet.AddStringId`. That way you don't need to get the packet data and its id yourself anymore.